README.md

```diff
@@ -41,33 +41,33 @@ Currently the storage-broker is creating a slow stream of auctions, so you might
 
 - PayloadCid (cid)
 - PieceCid (cid)
 - DealSize (int64 in bytes)
 - VerifiedDeal (bool)
 
-To initiate the storage of the new payload, the SB creates an *auction*. The *auction* is published to all connected miners to notify them that a new payload is awaiting storage on Filecoin. 
+To initiate the storage of the new payload, the SB creates an *auction*. The *auction* is published to all connected storage providers to notify them that a new payload is awaiting storage on Filecoin. 
 
 To understand better what this is about, consider the following chat:
 
-- [Storage Broker]: "Hey miners, there's a new Auction for a dataset with size X, PayloadCid X, PieceCid X, and PieceSize X. This data will be created with a verified deal. Who's interested?"
+- [Storage Broker]: "Hey storage providers, there's a new Auction for a dataset with size X, PayloadCid X, PieceCid X, and PieceSize X. This data will be created with a verified deal. Who's interested?"
 - [Miner-A]: "Me! Here's my Bid: price-per-epoch=0.0001FIL and I promise to accept a deal proposal with *DealStartEpoch=XXXX".*
 - [Miner-B]: Hey, me too! My price is 0.00000001FIL, and I promise to accept a deal proposal with DealStartepoch=YYYYY".
 - [Miner-C]: Uhm, I'm too busy now... I won't bid here and just let this pass. Maybe in the next auction!
 
-Miners will send *bids* to show their interest in participating in the auction. Bids provide the SB multiple pieces of information (i.e. price, start epoch, etc) about the miner's storage intent. Unlike the chat based auction, miners connected to the SB can configure bidding to happen automatically as fast as the miner's infrastructure can handle. 
+Miners will send *bids* to show their interest in participating in the auction. Bids provide the SB multiple pieces of information (i.e. price, start epoch, etc) about the storage provider's storage intent. Unlike the chat based auction, storage providers connected to the SB can configure bidding to happen automatically as fast as the storage provider's infrastructure can handle. 
 
-The SB collects all bids from connected miners and chooses one or multiple winners. The winning algorithm is still under development, but you can expect that the following facts will help winning an auction:
+The SB collects all bids from connected storage providers and chooses one or multiple winners. The winning algorithm aims to maximize deal success rate for storage clients, while being fair to all storage providers, big or small, new or established. It is open and subject to continuous evolving based on community feedback. Anyone can join the biweekly [Auction Governance meeting](https://textile.notion.site/Auction-Governance-2eb1acae8a204b6e8dbf72752255a008) to contribute. As a rules of thumb, you can expect that the following facts will help winning an auction:
```

Comment:
The main change. Prefer to not be very specific about the algorithm to avoid too frequent out-of-sync, but communicate what the miners may care about. Corrections and rephrases are welcome!

Reply:
Agree. Also later we can add here public dashboards of winners and some end results of goals we have.